README.md

```diff
@@ -52,16 +53,17 @@ JupyterHub can be installed with pip:
 
     pip3 install jupyterhub
 
-If the `pip3 install .` command fails and complains about `lessc` being unavailable, you may need to explicitly install some additional javascript dependencies:
+If the `pip3 install` command fails and complains about `lessc` being unavailable, you may need to explicitly install some additional JavaScript dependencies:
 
     npm install
 
-If you plan to run notebook servers locally, you may also need to install the IPython notebook:
+If you plan to run notebook servers locally, you may also need to install the
+Jupyter ~~IPython~~ notebook:
 
-    pip3 install "ipython[notebook]"
+    pip3 install jupyter
```

Comment:
@minrk Please double check this. I wasn't sure if we wanted to change this. And, if so, ``pip3 install jupyter`` or ``pip3 install notebook`` or ???